src/ontology/mondo-edit.obo

```diff
@@ -230447,7 +230445,7 @@ xref: OMIM:300612 {source="DOID:0060829", source="Orphanet:3056", source="MONDO:
 xref: OMIM:300706 {source="MONDO:equivalentObsolete", source="DOID:0060811", source="Orphanet:85328", source="Orphanet:85328/e"}
 xref: OMIM:309590 {source="Orphanet:3056", source="MONDO:equivalentTo"}
 xref: Orphanet:3056 {source="DOID:0060829", source="MONDO:equivalentObsolete", source="OMIM:300612"}
```

Comment:
This ([`xref: Orphanet:3056`](https://www.orpha.net/en/disease/detail/3056)) also has the same statement:
```
This entity has been excluded from the Orphanet nomenclature of rare diseases and moved to [Non-specific syndromic intellectual disability](https://www.orpha.net/en/disease/detail/528084)
```